kvbc/src/st_reconfiguration_sm.cpp

```diff
@@ -33,6 +34,18 @@ uint64_t StReconfigurationHandler::getStoredBftSeqNum(BlockId bid) {
   return sequenceNum;
 }
 
+uint64_t StReconfigurationHandler::getStoredEpochNumber(BlockId bid) {
+  auto value =
+      ro_storage_.get(kvbc::kConcordInternalCategoryId, std::string{kvbc::keyTypes::reconfiguration_epoch_key}, bid);
```

Comment:
The way of key generation may differ by storage impl, so it's better to use sort of `generateEpochKey()` at the interface level. Take a look at key manipulators.